spec/acceptance/sensu_asset_spec.rb

```diff
@@ -7,9 +7,12 @@
       pp = <<-EOS
       include ::sensu::backend
       sensu_asset { 'test':
-        url     => 'http://example.com/asset/example.tar',
-        sha512  => '4f926bf4328fbad2b9cac873d117f771914f4b837c9c85584c38ccf55a3ef3c2e8d154812246e5dda4a87450576b2c58ad9ab40c9e2edc31b288d066b195b21b',
-        filters => ['System.OS==linux'],
+        url      => 'http://example.com/asset/example.tar',
+        sha512   => '4f926bf4328fbad2b9cac873d117f771914f4b837c9c85584c38ccf55a3ef3c2e8d154812246e5dda4a87450576b2c58ad9ab40c9e2edc31b288d066b195b21b',
+        filters  => ['System.OS==linux'],
+        metadata => {
```

Comment:
Ugh, you have to specify this?

Reply:
Don't have to, but I wanted to test that it works when specified.

Reply:
Gotcha! Thought they required it which seemed crazy. I get it know.